CIP-0035/README.md

```diff
@@ -0,0 +1,302 @@
+| Adding cost model parameters                                               | HF                 | All nodes must recognize the new parameter.                                                         |
+| Removing cost model parameters                                             | LV                 | Old scripts will require all the old parameters.                                                    |
+| Improving real performance                                                 | SU                 |                                                                                                     |
+| Regressing real performance                                                | SU                 | _Must_ strictly follow a regression in model performance.[^4]                                       |
+
+[^1]: See "Are backwards-compatible binary format changes really safe?".
+[^2]: The binary format change is backwards compatible unless it breaches the limit of how many builtin functions or types can be encoded, in which case that must be changed, forcing a new LV.
+[^3]: See "Why do performance changes require extra steps?".
+[^4]: See "Why do performance changes require extra steps?".
+
+## Specification
+
+### Changes that require a CIP
+
+Changes that require a CIP do not have to each be in an individual CIP, they can be included in batches or in other CIPs. 
+So, for example, a single CIP could propose multiple new builtin functions, or a CIP proposing a change to the ledger could also propose a change to the ledger-script interface.
+
+The requirement to propose a change via a CIP is, as all CIPs are, advisory. 
+In exceptional circumstances or where swift action is required, we expect that changes may still be made without following this process. 
+In such circumstances, a retrospective CIP SHOULD be made after the fact to record the changes and the rationale for them.
+
+#### Additions to the Plutus Core Builtins
+
+Proposals for additions to the set of Plutus Core builtins SHOULD be included in a CIP and should follow both the usual CIP process and the process laid out in this document.
+
+In order to move to Draft status, it MUST include:
+- In the Specification:
+    - Names and types/kinds for the new functions or types.
+    - A source for the implementation (e.g. a library which can be linked against); or a generic description of the functionality which is implementable in any programming language.
+    - In the Rationale:
+    - An argument for the utility of the new builtins.
+
+It SHOULD also include:
+- In the Rationale
+    - Examples of real-world use cases where the new additions would be useful.
+    - A comparison with an implementation using the existing features, and an argument why the builtin is preferable (e.g. better performance).
```

Comment:
Who decides that the requirements to move to Draft have been met?

Reply:
Thanks for clarifying this: an inline link would help make that clearer in the text.